src/Pscid/Options.purs

```diff
@@ -121,18 +156,19 @@ buildOptions
   → Eff (fs ∷ FS | e) PscidOptions
 buildOptions port testAfterRebuild includes outputDirectory censor = do
   defaults ← unwrap <$> mkDefaultOptions
-  let sourceDirectories =
-        if null includes
-        then defaults.sourceDirectories
-        else filter (not null) (split (Pattern ";") includes)
+  let includesArr = filter (not null) (split (Pattern ";") includes)
+      sourceDirectories = defaults.sourceDirectories <> includesArr
       censorCodes = filter (not null) (split (Pattern ",") censor)
+      buildCommand = setCommandIncludes includesArr defaults.buildCommand
+      testCommand = setCommandIncludes includesArr defaults.testCommand
+
   pure (wrap { port: fromNumber (readInt 10 port)
              , testAfterRebuild
              , sourceDirectories
              , censorCodes
-             , buildCommand: defaults.buildCommand
+             , buildCommand: buildCommand
```

Comment:
If a variable with the same name as the field is in scope (like it is here in this PR) you can use a _record pun_ and avoid writing the same name twice.

`, buildCommand: buildCommand` -> `, buildCommand`

Reply:
Done.